src/express/common/milestone-utils.js

```diff
@@ -0,0 +1,396 @@
+  }
+
+  let response = false
+  await Promise.all(tasks)
+    .then(() => {
+      response = true
+    })
+    .catch((error) => {
+      console.log('📍Unable to remove peers, error:', error)
+    })
+
+  return response
+}
+
+export async function addPeers(ip, peers) {
+  const tasks = []
+  for (let i = 0; i < peers.length; i++) {
+    const command = `~/go/bin/bor attach ~/.bor/data/bor.ipc --exec "admin.addPeer('${peers[i]}')"`
+    tasks.push(runSshCommand(ip, command, maxRetries))
+  }
+
+  let response = false
+  await Promise.all(tasks)
+    .then(() => {
+      response = true
+    })
+    .catch((error) => {
+      console.log('📍Unable to add peers, error:', error)
+    })
+
+  return response
+}
+
+export async function getPeerLength(ip) {
+  const command =
+    '/home/ubuntu/go/bin/bor attach ~/.bor/data/bor.ipc --exec "admin.peers.length"'
+  try {
+    const length = await runSshCommandWithReturn(ip, command, maxRetries)
+    return parseInt(length)
+  } catch (error) {
+    console.log('📍Unable to query peer length, error:', error)
+  }
+
+  return -1
+}
+
+export async function joinAllPeers(ips, enodes) {
+  let tasks = []
+  for (let i = 0; i < ips.length; i++) {
+    tasks.push(addPeers(ips[i], enodes))
+  }
+
+  await Promise.all(tasks).then((values) => {
+    if (values.includes(false) || values.includes(undefined)) {
+      console.log('📍Failed to add peers for rejoining clusters')
+      return false
+    }
+  })
+
+  // Validate by fetching peer length
+  tasks = []
+  for (let i = 0; i < ips.length; i++) {
+    tasks.push(getPeerLength(ips[i]))
+  }
+
+  let lengths
+  await Promise.all(tasks).then((values) => {
+    lengths = values
+  })
+
+  if (!lengths) {
+    await timer(100)
+    if (!lengths) {
+      return false
+    }
+  }
+
+  if (lengths.includes(-1)) {
+    console.log('📍Unable to query peer length, exiting')
+    return false
+  }
+
+  console.log('📍Peer length:', lengths)
+  return true
+}
+
+export async function createClusters(ips, enodes, split = 1) {
+  // `split` defines how clusters are created and which index to use to seperate nodes.
+  // e.g. for split = 1, clusters created would be of 1 and 3 nodes (nodes[:split], nodes[split:])
+  const tasks = []
+  const ips1 = ips.slice(0, split)
+  const ips2 = ips.slice(split)
+  for (let i = 0; i < ips1.length; i++) {
+    tasks.push(removePeers(ips1[i], enodes.slice(split)))
+  }
+  for (let i = 0; i < ips2.length; i++) {
+    tasks.push(removePeers(ips2[i], enodes.slice(0, split)))
+  }
+
+  let response = false
+  await Promise.all(tasks)
+    .then((values) => {
+      if (values.includes(false) || values.includes(undefined)) {
+        console.log('📍Unable to remove peers, exiting')
+      } else {
+        response = true
+      }
+    })
+    .catch((error) => {
+      console.log('📍Unable to remove peers, error:', error)
+    })
+
+  return response
+}
+
+export async function rejoinClusters(ips, enodes, split = 1) {
```

Comment:
`rejoinClusters` and `validateNumberOfPeers` are not used, right?
In case, shall we remove them?